pages/[id].js

```diff
@@ -0,0 +1,58 @@
+import { useRouter } from "next/router";
+import useSWR from "swr";
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function EntryPage() {
+  const router = useRouter();
+  const { isReady } = router;
+  const { id } = router.query;
+
+  const {
+    data: entry,
+    isLoading,
+    error,
+  } = useSWR(`/api/entries/${id ? id : ""}`, {
+    fallbackData: {},
+  });
+
+  if (!id) {
+    return null;
+  }
+
+  //console.log("id:", id);
+
+  //const entry = entries.find((item) => item[_id] === id);
+
+  if (!isReady || isLoading) return <h2>Loading...</h2>;
+  if (error) return <h2>Failed to load entry</h2>;
+
+  const formattedDate = new Date(entry.dateTime).toLocaleString("de-DE", {
+    dateStyle: "medium",
+    timeStyle: "short",
+  });
+
+  return (
+    <>
+      <GoBackLink href="/"> ⬅️ Back </GoBackLink>
+      <section>
+        <p>Type:</p>
+        {entry.emotion.map(({ _id, emotion }) => {
+          console.log("emotion", emotion);
+          return (
+            <span key={_id} className={emotion.toLowerCase()}>
+              {emotion}
+            </span>
+          );
+        })}
```

Comment:
since we solved the bug, you can remove the console log and use an implicit return here instead again (remove the curly brackets and the return keyword)